e2e-tests/portal/cypress/integration/journeys/checker/cover-end-date/deal.js

```diff
@@ -0,0 +1,122 @@
+const mockDeal = require('../../../../fixtures/deal');
+const CONSTANTS = require('../../../../fixtures/constants');
+const { nowPlusDays } = require('../../../../support/utils/dateFuncs');
+
+const date = new Date();
+const datePlusDay = nowPlusDays(1);
+
+const deal = {
+  ...mockDeal,
+  mockFacilities: [
+    {
+      type: CONSTANTS.FACILITY.FACILITY_TYPE.BOND,
+      createdDate: date.valueOf(),
+      bondIssuer: '',
+      bondType: 'Bid bond',
+      facilityStage: CONSTANTS.FACILITY.FACILITY_STAGE.ISSUED,
+      hasBeenIssued: true,
+      'requestedCoverStartDate-day': '',
+      'requestedCoverStartDate-month': '',
+      'requestedCoverStartDate-year': '',
+      'coverEndDate-day': '06',
+      'coverEndDate-month': '01',
+      'coverEndDate-year': '2021',
+      name: '1234',
+      bondBeneficiary: '',
+      guaranteeFeePayableByBank: '18.0000',
+      updatedAt: Date.now(),
+      value: '1234.00',
+      currencySameAsSupplyContractCurrency: 'false',
+      currency: {
+        currencyId: 1,
+        id: 'AED',
+        text: 'AED - U.A.E. Dirham',
+      },
+      conversionRate: '5.5',
+      'conversionRateDate-day': '28',
+      'conversionRateDate-month': '04',
+      'conversionRateDate-year': '2022',
+      riskMarginFee: '20',
+      coveredPercentage: '30',
+      minimumRiskMarginFee: '',
+      ukefExposure: '370.20',
+      feeType: 'At maturity',
+      dayCountBasis: '365',
+      status: CONSTANTS.DEALS.DEAL_STATUS.UKEF_ACKNOWLEDGED,
+    },
+    {
+      type: CONSTANTS.FACILITY.FACILITY_TYPE.LOAN,
+      createdDate: date.valueOf(),
+      facilityStage: CONSTANTS.FACILITY.FACILITY_STAGE.CONDITIONAL,
+      hasBeenIssued: false,
+      ukefGuaranteeInMonths: '12',
+      guaranteeFeePayableByBank: '10.8000',
+      value: '123123.00',
+      currencySameAsSupplyContractCurrency: 'false',
+      interestMarginFee: '12',
+      coveredPercentage: '20',
+      minimumQuarterlyFee: '20',
+      ukefExposure: '24,624.60',
+      premiumFrequency: 'Monthly',
```

Comment:
Constants for `premiumFrequency` and `premiumType`.

Reply:
done